recipes/ua-parser/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{%set name = "ua-parser" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "e1f7cb16ccb68d6c93b19348e678500bec8cd1f692e49214dd38ca2d100ada9b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
To be pedantic, this isn't "from source" either. The tarball is still a compiled asset. To build from source, you'd need to pull source from git, then run `setup.py install`. Without doing that, this is still a compiled distribution.